Sources/Views/Products/Detail/ProductDetailWebViewTableViewCell.swift

```diff
@@ -63,7 +67,8 @@ func htmlFormattedString( font: UIFont, color: UIColor) -> String {
             let green = colorComponents[1] * 255
             let blue = colorComponents[2] * 255
 
-            return NSString(format: "%02X%02X%02X", Int(red), Int(green), Int(blue)) as String
+            let hexString = NSString(format: "%02X%02X%02X", Int(red), Int(green), Int(blue)) as String
+            return hexString
```

Comment:
why changing this to have a temp useless variable?

Reply:
We have an extension on UIColor that will produce a hex. I wanted to incorporate that here, so work in progress.

Reply:
Moved the function to an UIColor extension.